### PR TITLE
Updated Logo component: added fixedColor mode

### PR DESCRIPTION
**Pull request by cheina97**
Components updated:

- Logo (common)

This PR adds the fixedColor prop, which allows you to set a fixed color for the logo, which will not change according to the selected theme.
This feature can be useful in CornerButton component, where button color don't change according to the  current theme

### CODE REVIEW

**Review by QcFe — APPROVED** [review]
looks good, I would simply put `color` instead of `fixedColor` though

File: frontend/src/components/common/Logo/Logo.tsx [hunk 1]
```diff
@@ -5,15 +5,17 @@ import { ReactComponent as SvgLogo } from '../../../assets/logo.svg';
...
   widthPx?: number;
   className?: string;
+  color?: string;
 }
 
 const Logo: FC<ILogoProps> = ({ ...props }) => {
-  const { widthPx, className } = props;
+  const { widthPx, className, color } = props;
   return (
     <div className={'flex items-center justify-center m-0 p-0 ' + className}>
```
Comment (QcFe) [inline frontend/src/components/common/Logo/Logo.tsx:14]: Are we sure we want to keep the logo with this div wrapper? (with such flex stuff too?)

**Review by QcFe — APPROVED** [review]
For me this modification is alright, not sure about the div though

File: frontend/src/components/common/Logo/Logo.tsx (hunk 1 above)
Comment (cheina97) [reply frontend/src/components/common/Logo/Logo.tsx]: I removed it. It seems work correctly without it.

**Pull request merged**